server/server.js

```diff
@@ -0,0 +1,13 @@
+var express = require('express');
+var cors = require('cors');
+var path = require('path');
+var bodyParser = require('body-parser');
+var app = express();
+
+app.use(cors());
+app.use(bodyParser.json());
+app.use(express.static('./public'));
+
+app.listen(4870, function () {
+    console.log('New Dev Server Listening on localhost:4870!');
+});
```

Comment:
Have you seen https://webpack.js.org/configuration/dev-server/ ? It's a dev server with a few extra features. You should try using that instead of building + committing your own development tool.